src/repl/repl_core2.ts

```diff
@@ -164,12 +164,13 @@ class ReplCore implements IRepl {
         // Load REPL history from file.
         if (fs.existsSync(historyFile)) {
             fs.readFileSync(historyFile)
-                .toString()
-                .split('\n')
-                .reverse()
-                .filter((line: string) => line.trim())
-                // tslint:disable-next-line:no-any
-                .map((line: string) => (repl as any).history.push(line));
+            .toString()
+            // Split on \n (linux) or \r\n (windows)
+            .split(/(\\n|\\r\\n)/)
```

Comment:
The .split is fine, but I wonder if there is a better, generally accepted way of doing this.